css/style.css

```diff
@@ -20,11 +20,11 @@ table.translations tr.preview.has-glotdict .original::before {
 }
 
 table.translations tr.preview.has-glotdict th+.priority+.original::before {
-    margin-left: calc(-.5em - 32px);
```

Comment:
![image](https://user-images.githubusercontent.com/65488419/161433067-436022c8-0a11-4371-b318-329ea4a5ac56.png)
For non-logged users this needs some adjustments.

Reply:
Gosh… 
it didn't take my whole code cause I fixed it too:
![image](https://user-images.githubusercontent.com/17084006/161434444-b2cf2690-2750-4fa1-b1eb-9590d59a894a.png)
Can you add it please?